package-lock.json

```
@@ -11598,8 +11598,8 @@
       }
     },
     "wet-boew": {
-      "version": "github:wet-boew/wet-boew#9f6a068a37f8df17e608759807a438a65b23440a",
-      "from": "github:wet-boew/wet-boew#v4.0.70",
+      "version": "github:wet-boew/wet-boew#94d604c345a32140f68b4fe09bf02bf88ca431ad",
```

Le hash devrait être:

```suggestion
      "version": "github:wet-boew/wet-boew#467e5c67d891211c383bd8d40d1cc715b2da8e5e",
```